src/Auth/AuthContainer.js

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+// import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import Header from 'grommet/components/Header';
+
+function AuthContainer({ children }) {
```

Comment:
Without `` export ``, it's impossible to test the `` AuthContainer `` itself, you can only test the "connected" `` AuthContainer ``, did I explain myself?